packages/core/src/api/get-all-items.ts

```diff
@@ -29,10 +33,59 @@ export async function getAllItems(postTypes: string[]): Promise<Items> {
   });
 
   let result: Items = [];
+  const archiveParams = await getArchiveStaticParams({ postTypes });
```

Comment:
looks like you added this to getAllItems, getAllItems just gets single items from specified post types so this would be the wrong place to add this!